.github/pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+## Description
+<!--
+Thank a lot for taking time to contribute to Intersmash <3!
+
+Please provide a description of what your PR does providing a link (if applicable) to the issue it fixes. It is
```

Comment:
Please provide a description of what your PR does, by providing a link (if applicable) to the issue it fixes. It is

I just added a comma and a "by"